### PR TITLE
Add Edit Delete An Event In Personal Calendar

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java [hunk 1]
```diff
@@ -119,11 +120,12 @@ public void goToRightMenuTaskEventFromWeekView(String name, selectDayOption opti
     if (date != null && date != "") {
       switch (optionDay) {
       case DETAILTIME:
-        evt.scrollBarToGetElement(By.xpath(ELEMENT_EVENT_TASK_DETAIL_DATE_WEEK_VIEW_ONE_DAY.replace("$name", name)
-                                                                                           .replace("$date", date)));
+        //evt.scrollBarToGetElement(By.xpath(ELEMENT_EVENT_TASK_DETAIL_DATE_WEEK_VIEW_ONE_DAY.replace("$name", name)
```
Comment (mgreau) [inline calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java]: if this code is not relevant anymore, you can delete it, we can use the git history to retrieve this code if needed.

File: calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java [hunk 2]
```diff
@@ -970,7 +972,8 @@ public void deleteEventTask(String name, selectViewOption view, selectDayOption
     $(ELEMENT_CONTEXT_MENU_DELETE).click();
     if (isVerify) {
       if (isEvent)
-        alert.verifyAlertMessage(ELEMENT_CONFIRM_DELETE_EVENT_MSG);
+      {alert.verifyAlertMessage(ELEMENT_CONFIRM_DELETE_EVENT_MSG);
+      $(ELEMENT_CONFIRM_DELETE_EVENT_MSG).waitUntil(Condition.disappears,Configuration.timeout);}
```
Comment (mgreau) [inline calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java]: Please format the code with eXo Formatter Settings http://developer.exoplatform.org/#id-formatter-settings

File: calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java (hunk 1 above)
Comment (ilyeskallala) [inline calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java]: ok

File: calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java (hunk 2 above)
Comment (ilyeskallala) [inline calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java]: done

**Pull request merged**